docs/getting-started.md

```diff
@@ -156,10 +154,10 @@ And open http://localhost:8080 in a browser. You should see a blue “Hello Worl
 
 ### Step 2: Include CSS for a component
 
-Now that you have webpack configured to compile Sass into CSS, let's include the Sass files for the Material Design button. First install the Node dependency:
+Now that you have webpack configured to compile Sass into CSS, let's include the Sass files for the Material Design button. First, install the Node dependency:
 
 ```
-npm install --save-dev @material/button
```

Comment:
This change should be reverted. --save-dev saves @material/button as a dev dependency, which is need to compile the SCSS and JS.

Reply:
if omitted saves to `dependencies`. It is my understanding the `devDeps` are for development environment only.

Reply:
> if omitted saves to `dependencies`. It is my understanding the `devDeps` are for development environment only.

npm install normally includes the dev dependencies. It is saved to devdependencies because it's required for modifying the code, but not for using the code in a website.

Reply:
`npm install` installs both devDeps and deps unless passed the production flag. devDeps should include packages for testing framework, build steps, etc. See docs below.

https://github.com/npm/npm/blob/2e3776bf5676bc24fec6239a3420f377fe98acde/doc/files/package.json.md#devdependencies